medicines/doc-index-updater/src/delete_manager/errors.rs

```diff
@@ -0,0 +1,23 @@
+use std::{error, error::Error, fmt};
+#[derive(Debug, Clone)]
+pub struct DocumentNotFoundInIndex {
+    content_id: String,
+}
+
+impl DocumentNotFoundInIndex {
+    pub fn for_content_id(content_id: String) -> DocumentNotFoundInIndex {
+        DocumentNotFoundInIndex { content_id }
+    }
+}
+
+impl fmt::Display for DocumentNotFoundInIndex {
```

Comment:
there's a shorthand for this by deriving from `thiserror::Error`
https://github.com/MHRA/products/blob/2b305e1f628b862aa63f1cee77e158c1ab657b98/medicines/doc-index-updater/src/create_manager/sftp_client.rs#L5-L9

Reply:
let's leave this out for now